app/src/main/java/com/example/korail_aos/presentation/ticket/TicketFragment.kt

```diff
@@ -5,14 +5,47 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
-import com.example.korail_aos.R
+import com.example.korail_aos.databinding.FragmentTicketBinding
+import com.google.android.material.tabs.TabLayoutMediator
 
 class TicketFragment : Fragment() {
+    private var _binding: FragmentTicketBinding? = null
+    private val binding: FragmentTicketBinding
+        get() = requireNotNull(_binding) { "FragmentTicketBinding" }
+    private val information =
+//        arrayListOf(getString(R.string.ticket_verify), getString(R.string.ticket_pass) (???)
+        arrayListOf("승차권", "정기권⋅패스")
+
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        return inflater.inflate(R.layout.fragment_ticket, container, false)
+        _binding = FragmentTicketBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        setViewPager()
+        super.onViewCreated(view, savedInstanceState)
+    }
+
+    private fun setViewPager() {
+        val adapter = TicketVPAdapter(this)
+        binding.vpTicketVerifyViewpager.adapter = adapter
+        binding.vpTicketVerifyViewpager.run {
+            isUserInputEnabled = false
+        }
```

Comment:
```suggestion
        binding.vpTicketVerifyViewpager.apply{
            adapter = adapter
            isUserInputEnabled = false
        }
```
이런 식으로 단축할 수 있을 것 같습니다